### PR TITLE
Feat: About component

### PR DESCRIPTION
**Pull request by olegdolenchuk**

### CODE REVIEW

**Comment by braindrain9** [comment]
one minor - please decrease this purple svg icon size on the image for tablets

Screenshot 2020-05-03 at 19 39 57

**Comment by braindrain9** [comment]
also, this green box is missing from the mocks:

Screenshot 2020-05-03 at 19 41 55

File: src/app/pages/HomePage/About/component.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  <AboutWrapper>
+    <AboutImgWrapper>
+      <img src={AboutImg} alt="" />
+      <ProjectsVerifiedWrapper>
+        <ProjectsVerifiedIcon />
+      </ProjectsVerifiedWrapper>
+    </AboutImgWrapper>
+    <AboutTextWrapper>
+      <H2>
+        Mеценат -
```
Comment (braindrain9) [inline src/app/pages/HomePage/About/component.tsx:21]: please use https://react.i18next.com/latest/trans-component methodic here

File: src/app/pages/HomePage/About/component.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <span>
+          це <OverlayShapeCIcon />
+        </span>
+        допомога
```
Comment (braindrain9) [inline src/app/pages/HomePage/About/component.tsx:25]: please use https://react.i18next.com/latest/trans-component methodic here

**Comment by braindrain9** [comment]
please update screenshots

File: public/locales/uk/translation.json [hunk 3]
```diff
@@ -26,6 +26,10 @@
     "DESCRIPTION": "Ви можете знайти проект, який підходить вам ідеально, і дізнатися, як ваша компанія може підтримати його.",
     "HEADING": "Соціальні та благодійні ініціативи України"
   },
+  "about": {
+    "TEXT": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum eu fugiat nulla pariatur.",
+    "HEADING": "Mеценат - <1>це <1><0></0></1></1> допомога"
```
Comment (braindrain9) [inline public/locales/uk/translation.json:31]: HEADING should go before TEXT (alphabetical order)

File: public/locales/uk/translation.json [hunk 4]
```diff
@@ -26,6 +26,10 @@
     "DESCRIPTION": "Ви можете знайти проект, який підходить вам ідеально, і дізнатися, як ваша компанія може підтримати його.",
     "HEADING": "Соціальні та благодійні ініціативи України"
   },
+  "about": {
```
Comment (braindrain9) [inline public/locales/uk/translation.json:29]: please keep in alphabetical order. "about" should go before "footer"

File: src/shared/shapes/RectangleOvalToRight.tsx [hunk 5]
```diff
@@ -3,26 +3,35 @@ import React from 'react';
...
+      viewBox="0 0 170 194"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
       <path
         fillRule="evenodd"
         clipRule="evenodd"
         d="M0 0H170V194H0C0 194 11.1027 119.828 11.1027 81.573C11.1027 43.3184 0 0 0 0Z"
-        fill="#FFD167"
+        fill={fill ? fill : '#FFD167'}
```
Comment (braindrain9) [inline src/shared/shapes/RectangleOvalToRight.tsx:34]: could be simplified to `fill || Colors.YELLOW_DARK`

**Comment by braindrain9** [comment]
couple of minors, otherwise looks great 👍

File: src/shared/shapes/RectangleOvalToRight.tsx (hunk 5 above)
Comment (olegdolenchuk) [reply src/shared/shapes/RectangleOvalToRight.tsx]: Done!

File: public/locales/uk/translation.json (hunk 4 above)
Comment (olegdolenchuk) [reply public/locales/uk/translation.json]: Done!

File: public/locales/uk/translation.json (hunk 3 above)
Comment (olegdolenchuk) [reply public/locales/uk/translation.json]: Done!

**Pull request merged**